src/Symfony/Component/HttpKernel/EventListener/LocaleListener.php

```diff
@@ -75,6 +79,7 @@ private function setRouterContext(Request $request)
     public static function getSubscribedEvents()
     {
         return array(
+            KernelEvents::REQUEST => array(array('onKernelRequestEarly', 100)),
```

Comment:
This is broken. You cannot have the same key twice in an array. What you need is multiple elements in the array of `KernelEvents::REQUEST`